hubmap/frontend/src/pages/DetailsPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+const fakeMetadata = {
+  title: 'My nifty dataset',
+  'date-published': '2020-01-01',
+  authors: ['Austen, Jane', 'Basho', 'Carroll, Lewis'],
+  experiments: [
+    // { name: 'Foucault\'s Pendulum', demonstrates: 'Earth\'s rotation' },
+    // { name: 'Millikan\'s Oil Drop', demonstrates: 'Electron charge' },
+  ],
+  credits: { Catering: 'Clover', 'Dolly Grip': 'ABC', Gaffer: 'XYZ' },
+};
+
+function Details(props) {
+  const { metadata } = props;
+  return (
+    <table>
+      {
+        Object.entries(metadata).map(
+          ([key, value]) => {
+            let renderedValue = value;
+            if (Array.isArray(value)) {
+              renderedValue = '[array]';
+            } else if (typeof value === 'object') {
+              renderedValue = '[object]';
```

Comment:
if the value is an object: why not flatten the object's values and identify any other recursions and just display like other text (separated by comma ) ?